website/docs/r/integration_gcp_pub_sub_audit_log.html.markdown

```diff
@@ -0,0 +1,63 @@
+---
+subcategory: "Cloud Account Integrations"
+layout: "lacework"
+page_title: "Lacework: lacework_integration_gcp_pub_sub_audit_log"
+description: |-
+  Create and manage GCP Pub Sub Audit Log integrations
+---
+
+# lacework\_integration\_gcp\_pub\_sub\_audit\_log
+
+Use this resource to configure an [GCP Pub Sub Audit Log integration](https://docs.lacework.com/category/gcp-pub-sub-audit-log-integrations) to analyze GCP Pub Sub audit logs.
+
+## Example Usage
+
+```hcl
+resource "lacework_integration_gcp_pub_sub_audit_log" "account_abc" {
+	name             = "account ABC"
+	project_id       = "ABC-project-id"
+	subscription     = "projects/ABC-project-id/subscriptions/example-subscription"
+	topic_id         = "projects/ABC-project-id/topics/example-topic"
+	integration_type = "PROJECT"
+	credentials {
+		client_id      = "123456789012345678900"
+		client_email   = "gcp-al-pub-sub-test@techally-test.iam.gserviceaccount.com"
+		private_key_id = "1234abcd1234abcd1234abcd1234abcd1234abcd"
+		private_key    = "-----BEGIN PRIVATE KEY-----\n ... -----END PRIVATE KEY-----\n"
+	}
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The GCP Audit Trail integration name.
+* `organization_id` - (Optional) The organization ID. Required if `integration_type` is set to `ORGANIZATION`.
```

Comment:
nit: I think we sort required params first. Ignore if this is not the case on other resources markdown

Reply:
Seems to be a mixture